learnify/web/src/pages/SignUpForm.js

```diff
@@ -43,10 +43,41 @@ function SignUpForm() {
 
     const navigate = useNavigate();
 
-    const onSubmit = (data, event) => {
-        console.log(data, event);
+    const [message, setMessage] = useState("");
+    
+    const registerUser = async (username, email, password) => {
+        await fetch("http://3.75.151.200:3000/auth/signup", {
```

Comment:
We can put this URL into a .env file and use it from that instead of using it statically on the code since we will need to use this value several times on the front-end project. Also, the backend will deploy the project several times and probably change the URL.

Reply:
probably we will use it like;
await fetch("{process.env.BACKEND_URL}/auth/signup"